client/my-sites/checkout/src/components/payment-method-logos.js

```diff
@@ -3,19 +3,19 @@ import styled from '@emotion/styled';
 export const PaymentMethodLogos = styled.span`
 	display: flex;
 	flex: 1;
-	text-align: right;
+	flex-wrap: wrap;
+	gap: 5px;
+	text-align: end;
```

Comment:
This should resolve the PayPal logo issue. Changed to a more responsive value to handle RTL better.